### ISSUE TITLE
audit alignment of displayed user data and content

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#3781]
### Observed behavior

Sometimes we use `text-align` on user data and content. This over-rides `dir="auto"`, which results in content and text not being properly localized.

### Expected behavior

* No displayed user data or content should have `right` or `left` set for the `text-align` property (`center` is OK).
* All displayed user data and content should have `dir="auto"`
* `dir="auto"` should not generally be set on 'wrapper' nodes - it should be as close to the text as possible
* Translated "app text" does not generally need to have `dir="auto"`

| Incorrect | Correct |
|--|--|
| ![image](https://user-images.githubusercontent.com/2367265/41183996-500f2f12-6b32-11e8-99ce-1f6ea6df4912.png) | ![image](https://user-images.githubusercontent.com/2367265/41183934-09875b0a-6b32-11e8-80a6-08ae6b2386de.png) |

### User-facing consequences

Mixed LTR/RTL text is harder to read

### Context

0.10.0

**Comment by Priyaraj17** [comment]
Is this issue open? May I work on it?

**Comment by nucleogenesis** [comment]
@Priyaraj17 sure that'd be much appreciated. If you have any questions please let us know.

**Comment by Priyaraj17** [comment]
Sure, I will get started. Can you tell me how to reproduce this issue?

**Comment by nucleogenesis** [comment]
Yes - @Priyaraj17 this issue is a matter of ensuring that we don't use the `text-align` property in our CSS with the `right` and `left` values.

Resolving the issue would involve:

- [ ] Finding all instances of `text-align` in our CSS (both inline CSS and the stuff between `<style>` tags) and making sure that they do not assign the value of `right` or `left`.
- [ ] If you make a change, ensure that the content affected by that CSS change displays correctly in RTL and LTR. If it doesn't, you may need to add `dir="auto"` attribute to the HTML.

When you create a PR - and as you go - it would be helpful during review if you provide screenshots.

**Comment by Priyaraj17** [comment]
Thank you. I will start ASAP.

**Comment by indirectlylit** [comment]
Just to clarify a subtle but important distinction:

Using `text-align` with `right` and `left` is entirely acceptable for page layout. These values should be written assuming an LTR layout, and they will automatically be swapped when viewed in an RTL language using [RTLCSS](https://rtlcss.com/).

This issue is just referring to user-generated text, for example names and descriptions of content, usernames, facility names, etc. Specifically this type of text should not use `text-align`, and should instead use `dir="auto"`.

For more information see https://kolibri-dev.readthedocs.io/en/develop/i18n.html#text-alignment

**Comment by muhammadalics** [comment]
Hi @Priyaraj17 are you working on this issue? If not, can I give it a shot?

**Comment by Priyaraj17** [comment]
Hey @muhammadalics I am working on this. If I face some issues, I will definitely ask for your help. Thanks

**Comment by Priyaraj17** [comment]
I have made a few changes, I am still working on some of them. Please can someone review this PR?

**Comment by indirectlylit** [comment]
@Priyaraj17 thank you for your effort but the PR cannot be merged as submitted. Please see my note here: https://github.com/learningequality/kolibri/pull/7996#issuecomment-818048313

**Comment by Priyaraj17** [comment]
@indirectlylit Thank you. I will keep everything in mind and will work on creating a different PR for this issue.

**Comment by avishmehta68710** [comment]
Hi @Priyaraj17 would you like to work in collab mode ????

**Comment by Priyaraj17** [comment]
@avishmehta68710 yeah sure. Infact we have to go through a lot of code for this issue. We can work in collab.
